Added  queues & stacks

ayaan-qadri opened the pull request:
<!--
Hello! Thank you for contributing to Code Thesaurus!

This template can help you fill out a pull request with all the information 
needed to review it. The text between these exclamation tags are comments 
and won't show up on the pull request. The text after ## are headers. Please
leave the headers unless otherwise noted and follow the instructions to 
fill out each section with as much information as you can to assist the 
reviewer!
-->


## What GitHub issue does this PR apply to?

 <!--
   If you're working on an existing issue, replace "xxxx" below with the issue 
   number. You can change it to say only "Closes #", "Fixes #", or "Resolves #".
   Don't add the word "issue" to it otherwise it won't link correctly.

   If you got here via the edit buttons on the website and/or your changes 
   don't relate to an issue, feel free to delete "Resolves #" and put "None", 
   or note why you chose to make a PR!
   -->

Resolves #586 


## What changed and why?

   Added stack and queue in lua


## (If editing Django app) Please add screenshots

   <!--
   If this doesn't apply, you can delete this header and section.
   If it's applicable, please copy screenshots, then paste them below.
   GitHub will paste images as a line of text as: ![image](url)
   You can preview it with the "Preview" button above.
   -->


## Checklist

   <!-- 
   Each - [ ] below is a checkbox that will show up on the PR.
   Either add an X inside the [X], or submit the PR and click the checkboxes.
   -->

- [X] I claimed any associated issue(s) and they are not someone else's
- [X] I have looked at documentation to ensure I made any revisions correctly
- [X] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes


## Any additional comments or things to be aware of while reviewing?

   <!-- Please replace this line with any extra information that's helpful -->

ayaan-qadri committed:
Added  queues & stacks

ayaan-qadri commented:
Hey @geekygirlsarah,
can you pls check PR and lemme know if there any changes need to be done.

geekygirlsarah commented on web/thesauruses/lua/5/queues_stacks.json:
```diff
@@ -0,0 +1,312 @@
+    },
+    "stack_data_type": {
+      "name": "Stack data type name",
+      "code": [
```
This should probably be `not-implemented` and the code should be a comment instead.

geekygirlsarah reviewed (changes requested):
Change the one thing and I'll merge this!

ayaan-qadri committed:
Update queues_stacks.json 

Updated file as per https://github.com/codethesaurus/codethesaur.us/pull/705#pullrequestreview-1751634586

geekygirlsarah reviewed (approved):
I think I dropped the ball on re-reviewing this, sorry about that!

I'll merge it in. Thank you for the change!
